source/MissionAction.cpp

```diff
@@ -112,6 +112,12 @@ void MissionAction::Load(const DataNode &node, const string &missionName)
 		// The legacy syntax "outfit <outfit> 0" means "the player must have this outfit installed."
 		else if(key == "outfit" && child.Size() >= 3 && child.Token(2) == "0")
 		{
+			// Do not support this syntax for NPC actions, since they were created after it was deprecated.
+			if(isNPCAction)
+			{
+				child.PrintTrace("Error: unsupported use of \"outfit\" inside an NPC action.");
+				continue;
+			}
```

Comment:
It doesn't seem like there's any actual harm done in allowing this action to be used on an NPC, so I'd just remove this warning.